# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini
# Commit: a16d763f2866
# Full Hash: a16d763f2866a1c44e67b85a66fa4503b5c764ac
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-09-13 09:25:56
# Regressor Bug: 1287054
# File Overlap Count: 1
# Description:
#   Bug 1287054 part 1 - support rem, ch, ic and cap units for non-CSS lengths r=emilio,jgilbert
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D180618
# ==============================================================================

diff -r ccfee1091860 -r a16d763f2866 testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini
--- a/testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini	Tue Sep 12 20:57:49 2023 +0000
+++ b/testing/web-platform/meta/svg/types/scripted/SVGLength-ic.html.ini	Tue Sep 12 21:10:54 2023 +0000
@@ -1,8 +1,3 @@
 [SVGLength-ic.html]
   expected:
     if (os == "android") and fission: [OK, TIMEOUT]
-  [ic unit in SVGLength]
-    expected: FAIL
-
-  [Convert back to ic from new user unit value]
-    expected: FAIL